Sources/MessagingPushAPN/RichPush/MessagingPush+NotificationServiceExtension.swift

```diff
@@ -40,6 +42,18 @@ public extension MessagingPush {
             return false
         }
 
+        // If cioHandleBadge exists and value is true
+        if let cioHandleBadge = pushContent.cioHandleBadge, cioHandleBadge  {
+            
+            let badgeCount = store.badgeCount ?? 0
```

Comment:
What is the use case for getting the stored badge count from global storage? What is the app has a badge count already or the global storage value is out-of-date?